src/main/resources/application.yaml

```diff
@@ -3,8 +3,14 @@ server:
 
 management:
   endpoints:
+    exposure:
+      include: '*'
     web:
       base-path: /
+  endpoint:
+    health:
```

Comment:
is this part of the client integration ?

Reply:
no, once i get further along i can move it to a separate PR, but its a standard that should be there otherwise you don't know why the health check is failing